src/Projections/Windows.UI.Xaml/ABI.Windows.System/DispatcherQueueProxyHandler.cs

```diff
@@ -144,14 +143,64 @@ private static ref readonly Guid IID_IDispatcherQueueHandler
             }
         }
 
+        /// <summary>The IID for <c>IUnknown</c> (00000000-0000-0000-C000-000000000046).</summary>
+        private static ref readonly Guid IID_IUnknown
```

Comment:
We should be able to use `WindowsRuntime.InteropServices.WellKnownInterfaceIIDs` for these two IIDs, right?